### PR TITLE
Don't filter out any water features

### PR DESCRIPTION
**Pull request by luqmaan**
image

image

### CODE REVIEW

File: frontend/src/components/Shared/Map/style/style.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
             "collapsed": false
           }
         },
-        "openmaptiles:version": "3.x",
+        "fmaptiles:version": "3.x",
```
Comment (luqmaan) [inline frontend/src/components/Shared/Map/style/style.json]: This line changed after I ran `npm run build-map-style`. Not sure why.

Comment (briaguya) [reply frontend/src/components/Shared/Map/style/style.json]: That is very odd, build.js in Map/style/scripts doesn't appear to touch that file.

File: frontend/src/components/Shared/Map/style/style.json [hunk 2]
```diff
@@ -99,20 +99,6 @@
...
-                    "$type",
-                    "LineString"
-                ],
-                [
-                    "in",
-                    "class",
-                    "canal",
-                    "river"
-                ]
-            ],
```
Comment (briaguya) [inline frontend/src/components/Shared/Map/style/style.json]: Just want to make sure this is the only layer we have with the waterway source layer, I'm not sure if we have an incorrectly configured "canals are a different color" type layer in there.

Comment (luqmaan) [reply frontend/src/components/Shared/Map/style/style.json]: Yeah there's no filtering in the water stuff: https://github.com/cityofaustin/ctxfloods/blob/44910295564a39f10b6519a7a32bf70dedd8f478/frontend/src/components/Shared/Map/style/style.json

**Pull request merged**